### DISCUSSION TITLE
developer-case-studies/fcpxml/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#352] [Website Discussions]
# developer-case-studies/fcpxml/

Written by Philip Hodgetts & Dr Gregory Clarke from Intelligent Assistance

https://fcp.cafe/developer-case-studies/fcpxml/

**Comment by marcobonini** [comment]
Why it's so difficult to translate a FCPXML to OZML?

**Reply by latenitefilms** [reply]
FCPXML and OZML are two completely different things.

OZML is also proprietary and undocumented.

**Comment by ioVideoMaker** [comment]
impressive dissertation... so you surely understand this thing that is driving me crazy:

why do I have ```<<video ref="r12" offset="11s" name="Custom" start="3600s" duration="10s">```

what's start?
the children of that event all refer to it to define their start, like ```<asset-clip ref="r13" lane="1" offset="3601s" ...```

(meaning it starts at 12s in my timeline, 11s of r12 event plus 1 second offset)

...ok....
...but why 3600s??????

thanks to anyone competent enough to make it clear!

Marco - ioVideoMaker
